sys-fs/dd-rescue/files/dd-rescue-1.99.11-musl-r2.patch

```diff
@@ -1,30 +1,135 @@
-From 69c3974670f5a8ee0f2258f10a9228b39025b464 Mon Sep 17 00:00:00 2001
-From: Thomas Deutschmann <whissi@gentoo.org>
-Date: Wed, 13 Dec 2017 01:02:12 +0100
-Subject: [PATCH] loff_t and __WORDSIZE includes for MUSL
-
-Rewrite of Justin Keogh's patch [Link 1] to fix build problems
-on ARM.
-
-Link 1: https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=5f5abc0f1b036921d6eb5b0f434c960ed280619f
-Fixes: https://bugs.gentoo.org/616364
----
- configure.ac | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
-diff --git a/configure.ac b/configure.ac
-index 7332da8..ef2016a 100644
+# Original patch was created by Thomas Deutschmann <whissi@gentoo.org>, but the
```

Comment:
Good. But could you keep the original Subject/"Rewrite ..."/"Link 1" and bug reference? Just keep lines 3-10 IMO (below what you already wrote).

Reply:
@thesamesam np, I've tried to preserve the original patch as much as possible, the only removed parts are removing changed files and after running scrub-patch